tests/ui/const-generics/bad-subst-const-kind.stderr

```diff
@@ -1,9 +1,23 @@
+error: the constant `13` is not of type `u64`
+  --> $DIR/bad-subst-const-kind.rs:13:24
+   |
+LL | pub fn test() -> [u8; <[u8; 13] as Q>::ASSOC] { todo!() }
+   |                        ^^^^^^^^ expected `u64`, found `usize`
```

Comment:
the compiler suggests using a `u64` as an array length

Reply:
yeah we have bad spans for this in general